app/src/renderer/js/editor.js

```diff
@@ -191,17 +244,17 @@ document.addEventListener('DOMContentLoaded', () => {
 
   inputHeight.addEventListener('keydown', handleKeyDown);
 
-  inputHeight.addEventListener('blur', function () {
+  inputHeight.addEventListener('blur', () => {
     this.value = this.value || (shake(this) && lastValidInputHeight); // Prevent the input from staying empty
   });
 
-  fps15Btn.addEventListener('click', function () {
+  fps15Btn.addEventListener('click', () => {
     this.classList.add('active');
     fpsMaxBtn.classList.remove('active');
     fps = 15;
   });
 
-  fpsMaxBtn.addEventListener('click', function () {
+  fpsMaxBtn.addEventListener('click', () => {
     this.classList.add('active');
```

Comment:
You can't use arrow function when `this` is used, like here.

Reply:
Oops. Didn't catch those.